test/e2e/specs/specs-playwright/wp-gutenboarding__happy-path.ts

```diff
@@ -69,11 +68,6 @@ describe( DataHelper.createSuiteTitle( 'Gutenboarding: Create' ), function () {
 		await gutenboardingFlow.selectPlan( 'Free' );
 	} );
 
-	it( 'See the Gutenberg editor', async function () {
-		const gutenbergEditorPage = new GutenbergEditorPage( page );
-		await gutenbergEditorPage.waitUntilLoaded();
-	} );
-
```

Comment:
What do you think of checking that "My Home" loads here, since that's the new destination after the /new site creation flow?

Reply:
Sure. Let me take a look!

Reply:
Quick question: would the text always be "Update your homepage"?

![image](https://user-images.githubusercontent.com/6549265/135346380-e6a8768f-5ee3-42e1-a9a5-db411ba4db84.png)

Reply:
I've included a step to check whether the URL with the schema `/home/<site-name>` has loaded prior to deleting the test account.

Reply:
I'm fairly sure that first task can vary depending on the particular site. The checklist is dynamic and some tasks can be auto completed.